### ISSUE TITLE
Questions about params setting

### ISSUE THREAD

**Issue by NUSLCC** [yufeiwang63/RL-VLM-F#17]
Hi Yufei,

Just want to ask about how to set the `num_seed_steps`, `num_unsup_steps`, `num_train_steps` in PEBBLE training. I set up a new environment with a robot manipulation task, like open a drawer, but did not get good results by using same configs. May I ask about your opinions on these parameters?

Thanks

**Comment by yufeiwang63** [comment]
Hi,

Can you talk a bit more about the task? We used the same values for these variables in PEBBLE training. 

num_seed_steps is the number of steps where we run with random actions to collect exploration trajectories.
num_unsup_steps is the number of steps for using the PEBBLE state-based exploration reward to facilitate exploration.
num_train_steps is the total RL training steps.

One thing we found helpful in our experiments is to remove the robot from the image when querying the VLM for the preference label, since VLMs are probably not trained on too much robot data.

**Comment by NUSLCC** [comment]
Hi Yufei, thank you for replying. I replaced the manipulation tasks from Meta-World with a franka panda pick-and-lift as my new task. I tried the num_seed_steps(1000), num_unsup_steps(9000), num_train_steps(1m) to test the performance, but somehow the robot is suffering from a local maximum (keep close to the cube and not pick it up). I changed the prompt to ensure the desire of lifting, but seems not helpful in 1m training steps.

Have you fine-tuned any other parameters? Thank you for the advice of making robot invisible, actually the meta-world manipulation tasks can be realized by making robot visible, based on latest VLM-models. 

Really Appreciate it.

Chenchen
